js/flightlog.js

```diff
@@ -680,6 +725,21 @@ function FlightLog(logData) {
                         }
                     }
 
+                    if(gps) {
+                        console.log("GPS");
+                        console.log("destFrame", destFrame);
+                        console.log("srcFrame", srcFrame);
+                        destFrame[fieldIndex++] = srcFrame[gps[0]];
+                        //destFrame[fieldIndex++] = 42069;
+                        destFrame[fieldIndex++] = srcFrame[gps[1]];
+                        destFrame[fieldIndex++] = srcFrame[gps[2]];
+                        destFrame[fieldIndex++] = srcFrame[gps[3]];
+                        destFrame[fieldIndex++] = srcFrame[gps[4]];
+                        destFrame[fieldIndex++] = srcFrame[gps[5]];
+                        destFrame[fieldIndex++] = srcFrame[gps[6]];
+                        console.log("destFrame", destFrame);
+                        console.log("srcFrame", srcFrame);
+                    }
```

Comment:
If you add the data in the P and I frames, I suppose this can be removed. The data is not calculated, has been filled in the other function.

Reply:
Agree, removed it.